operator/scripts/update-kcc-manifest/main.go

```diff
@@ -268,18 +270,79 @@ func main() {
 		log.Fatalf("error updating file %v", stableFilePath)
 	}
 
-	staleManifestDir := path.Join(operatorSrcRoot, "channels", "packages", "configconnector", currentVersion.Version)
-	log.Printf("removing stale manifest %v", staleManifestDir)
-	if err := os.RemoveAll(staleManifestDir); err != nil {
-		log.Fatal(fmt.Errorf("error deleting dir %v: %w", staleManifestDir, err))
+	channelDir := path.Join(operatorSrcRoot, "channels", "packages", "configconnector")
+	if err := dropStalePackages(channelDir); err != nil {
+		log.Fatalf("drop stale packages: %s", err)
 	}
-	staleManifestDir = path.Join(operatorSrcRoot, "autopilot-channels", "packages", "configconnector", currentVersion.Version)
-	log.Printf("removing stale manifest %v", staleManifestDir)
-	if err := os.RemoveAll(staleManifestDir); err != nil {
-		log.Fatal(fmt.Errorf("error deleting dir %v: %w", staleManifestDir, err))
+	autoPilotChannelDir := path.Join(operatorSrcRoot, "autopilot-channels", "packages", "configconnector")
+	if err := dropStalePackages(autoPilotChannelDir); err != nil {
+		log.Fatalf("drop stale packages: %s", err)
 	}
 }
 
+func dropStalePackages(pacakgesPath string) error {
+	// packagesPath := path.Join(operatorSrcRoot, "channels", "packages", "configconnector")
+
+	dirEntries, _ := os.ReadDir(pacakgesPath)
+	versionMap := map[int][]int{}
```

Comment:
Nit: it's probably easier to use either the filepath or a struct with info here.  The advantage of a struct is that you can more easily name the fields.